pkgs/by-name/op/openapi-python-client/package.nix

```
@@ -0,0 +1,80 @@
+{
+  lib,
+  stdenv,
+  darwin,
+  python3Packages,
+  fetchFromGitHub,
+  installShellFiles,
+  ruff,
+  testers,
+  openapi-python-client,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "openapi-python-client";
+  version = "0.21.4";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    inherit version;
+    owner = "openapi-generators";
+    repo = "openapi-python-client";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/v${version}";
```

Avoids tag/branch mismatch.

---

Changed this and updated to 0.21.5